data/quarg missions.txt

```diff
@@ -13,6 +13,7 @@ mission "First Contact: Quarg"
 	source
 		attributes "quarg"
 	on offer
+		log "Visited a Quarg spaceport. Spoke with one of them briefly."
```

Comment:
Seems a little short to me. I might include some information that we learn (regardless of our conversation decisions).